packages/sources/kaiko-state/src/transport/state.ts

```diff
@@ -73,11 +73,11 @@ export class KaikoStateTransport extends SubscriptionTransport<KaikoStateTranspo
 
     subscription.on('data', (response: StreamAggregatedStatePriceResponseV1) => {
       const cacheData = []
-      if (response.getAggregatedPriceEth().length > 0) {
+      if (response.getAggregatedPriceLst().length > 0) {
         cacheData.push(
           this._generateResponse(
-            'ETH',
-            response.getAggregatedPriceEth(),
+            response.getLstQuote().toUpperCase() == 'WETH' ? 'ETH' : response.getLstQuote(),
```

Comment:
Some DP return WETH some return ETH, we made the decision to quote everything under ETH

Reply:
FWIW in case anyone else is looking for it, this is all documented in the EA requirements doc

Is this ever expected to be `eth`?
Could we ever run into an dupe issue where we receive eg: btc-weth -> cache btc-ETH and receive btc-eth -> cache btc-eth since `quote` is not uppercased in `_generateResponse`?

Reply:
So in this case btc-weth will be overridden by btc-eth, let me ask DP

Reply:
Confirmed we are fine with override, but need to update the logic to be more generic

Reply:
Actually this logic is fine